ocs/exceptions.py

```diff
@@ -14,12 +14,11 @@ class UnexpectedBehaviour(Exception):
     pass
 
 
-# TODO: Delete once moved to pytest
-class UnknownStatusOfTestException(Exception):
+class ClassCreationException(Exception):
     pass
 
 
-class ClassCreationException(Exception):
+class LeftoversDetected(Exception):
```

Comment:
Somehow I am not convinced about this name. I feel following would be more readable.
1. ResourceLeftoversException()
2. GarbageLeftoversException().

Also Leftovers doesn't indicate the resources which deleted by test but not created, because there is nothing leftover there.

Reply:
I will change to ResourceLeftoversException().
But what about your "Also" part?